electronics/_kicad_library/Polarity.pretty/Polarity_Center_Positive_2mm_SilkScreen.kicad_mod

```diff
@@ -0,0 +1,169 @@
+(module Polarity_Center_Positive_2mm_SilkScreen (layer F.Cu) (tedit 0)

```

Comment:
I'm assuming these are "labels" or indicators for silkscreening.  Does it make sense to rename the Polarity.pretty library to make this more explicit?  Or does it not matter?  (The filenames themselves are good and sufficiently descriptive.)